hoomd/BondedGroupData.cuh

```diff
@@ -19,14 +19,14 @@ namespace hoomd
 //! Sentinel value
 const unsigned int GROUP_NOT_LOCAL = 0xffffffff;
 
-//! Storage for group members (GPU declaration)
-template<unsigned int group_size> union group_storage {
+    //! Storage for group members (GPU declaration)
+    template<unsigned int group_size> union group_storage {
     unsigned int tag[group_size]; // access 'tags'
     unsigned int idx[group_size]; // access 'indices'
     };
```

Comment:
This style of indentation doesn't seem consistent with what we enforce for all the other parts of hoomd. Did it get set this way on purpose, or do we need to fix some `clang-format` setting?

Reply:
@tommy-waltmann #1155 introduced many of these changes as well. The `conda-forge` build of `clang-format 12.0.0` formats the code one way and the `pypi` build of `clang-format 12.0.1` and `13.0.0` format it this new way.

I cannot find any `clang-format` setting that controls this. `clang-format` regularly introduces styling changes with every new release. It seems strange that such a change would result from `12.0.0` to `12.0.1`, but it also seems strange that the `pypi` build would format code differently from the `conda-forge` build.

If you have the time to invest, feel free to find and change the appropriate setting or find the corresponding change in `clang-format` and propose a fix upstream.